common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -18,13 +19,31 @@ fun warnLog(message: String) {
 }
 
 fun errorLog(message: String) {
-    if (Config.debugLogsEnabled) {
-        Log.e("[Purchases] - ERROR", message)
-    }
+    Log.e("[Purchases] - ERROR", message)
 }
 
 fun errorLog(error: PurchasesError) {
-    if (Config.debugLogsEnabled) {
-        Log.e("[Purchases] - ERROR", error.toString())
+    when (error.code) {
```

Comment:
I'm confused, what's this method doing?

Reply:
@aboedo this method produces backend error messages with emojis, instead of modifying the strings in `errors.kt`